lib/html_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'sinatra/base'
+
+module Sinatra
```

Comment:
I think you might have misunderstood my comment. I wasn't saying "you should change this". I was asking why it was like this. In general it's good to avoid simply cargo-culting code, even from official documentation. It's better to understand _why_ an example is a certain way. Is it's because it's the way you *need* to do something? (In this case that seems unlikely if it still works now that you've done it differently.)  With something like Sinatra, with two main modes of operation, perhaps an example is particularly geared towards one or the other? Or perhaps it's a convention that, whilst not strictly necessary, would be good to follow in the absence of any compelling reason why _not_ to. Or perhaps it's just one of many possible ways to achieve the same thing, and we're better sticking with something that more fits our existing codebase and approach. (We already have another Helper file. Introducing a second that works differently makes it a little more confusing for anyone reading the code, and harder for someone to know what to do when introducing another one. The two files don't necessarily need to be perfectly harmonised, but if they're different, we should be doing so knowingly.)

NB: when you _do_ take code from elsewhere, you should credit that.